dashboard/app/views/pd/workshop_mailer/teacher_enrollment_reminder.html.haml

```diff
@@ -48,8 +48,8 @@
   %p
     - if @workshop.course == Pd::Workshop::COURSE_CSF
       This is a great step towards bringing computer science to your classroom!
-    - else
```

Comment:
Note: This `else` clause had no body before, so it was effectively a no-op.  I'm not sure if that was a bug, but I'm preserving the existing behavior here.